documentation/asciidoc/topics/yaml/prometheus_service_monitor.yaml

```diff
@@ -3,26 +3,27 @@ kind: ServiceMonitor
 metadata:
   labels:
     k8s-app: prometheus
-  name: datagrid-monitoring <1>
-  namespace: infinispan-monitoring <2>
+  name: infinispan-monitoring # Names the service monitor.
+  namespace: infinispan-monitoring # Namespace for the service monitor.
 spec:
   endpoints:
-    - targetPort: 11223 <3>
-      path: /metrics <4>
+    - port: infinispan-adm
+      path: /metrics
       honorLabels: true
       basicAuth:
         username:
           key: username
-          name: basic-auth <5>
```

Comment:
Isn't removing the description technically a regression?

Reply:
I was suggesting to remove, but validates and suggest to keep them as is

Reply:
I'd say removing description for eg. path is ok as there it's standardized and non-cofigurable by user. `basic-auth` on the other side refers to a secret created in previous step and it may not be clear. Having a comment will draw an attention to the field and the need to set it correctly

Reply:
Good point @Crumby I'll add that back.